cpp/ql/src/Likely Bugs/Arithmetic/SignedOverflowCheck.qhelp

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+When checking for integer overflow, you may often write tests like
+<code>a + b &lt; a</code>.  This works fine if <code>a</code> or
+<code>b</code> are unsigned integers, since any overflow in the addition
+will cause the value to simply "wrap around."  However, using
+<i>signed</i> integers is problematic because signed overflow has undefined
+behavior according to the C and C++ standards.  If the addition overflows
+and has an undefined result, the comparison will likewise be undefined;
+it may produce an unintended result, or may be deleted entirely by an
+optimizing compiler.
+</p>
+</overview>
+<recommendation>
+<p>
+Solutions to this problem can be thought of as falling into one of two
+categories: (1) rewrite the signed expression so that overflow cannot occur
+but the signedness remains, or (2) rewrite (or cast) the signed expression
+into unsigned form.
+
+The table below lists various expressions where signed overflow may
+occur, along with proposed rewritings.  It should not be
+considered as exhaustive.
+</p>
+<table>
+  <tr>
+    <th>Original Construct</th>
+    <th>Alternate Construct(s)</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <td><tt><table>
```

Comment:
> Here are the errors from QHelp-Preview: https://gist.github.com/jbj/3cc7d8909268195858b5d29f76c05a3d

This will be tricky to fix, as I haven't figured out how to run `./target/jars/qhelpGithubPreview` on my local machine, outside of Jenkins.  If all else fails, I'll change the table to a bulleted list.

Reply:
> I'd be surprised if the qhelp schema allows a `<tt>` outside a `<table>`. What's the difference between `<tt>` and `<code>`?

It seems that `<tt>` is deprecated and `<code>` / `<kbd>`  / `<samp>` / `<var>`  are the replacements.

Reply:
It seems that `<tt>` has been deprecated in favor of `<code>`, `<kbd>`, `<samp>` and `<var>`, presumably for more fine-grained CSS control (at least that would be my guess).